### ISSUE TITLE
Update ini from 1.3.5 to 1.3.8.  

### ISSUE THREAD

**Issue by cah-briangantzler** [kaliber5/ember-bootstrap#1379]
unforntunately the fix https://github.com/kaliber5/ember-bootstrap/pull/1367 only changed the package.lock for ember boot straps dummy app. Anyone using ember-bootstrap is still getting this warning to update.

The below addons are going to need to be updated and them ember bootstrap depend on a new version of findup-sync, else every application using ember-bootstrap is going to have to include their own resolution to ini 1.3.8

=> Found "ini@1.3.5"
info Reasons this module exists
   - "ember-bootstrap#findup-sync#resolve-dir#global-modules#global-prefix" depends on it
   - Hoisted from "ember-bootstrap#findup-sync#resolve-dir#global-modules#global-prefix#ini"

**Comment by jelhan** [comment]
@cah-briangantzler Is a `yarn upgrade --pattern ember-bootstrap` upgrading `ini`? If not, did you tried to reinstall it with `yarn add ember-bootstrap`?

**Comment by cah-briangantzler** [comment]
I normally upgrade using `yarn upgrade-interactive --latest` and then choosing the several I want to upgrade. I would hope they would do the same thing. However running `yarn upgrade --pattern ember-bootstrap` did not change my yarn.lock or package.json at all.

It was my understanding that changes to ember-bootstraps yarn.lock would have no bearing on my applications yarn.lock as it is recalculated based off the packages that are brought in. If you look at the commit PRs commit referenced above, you will find it only changed ember-bootstraps yarn.lock. I would need to do in my application the same type of commit to get my applications yarn.lock to update.

**Comment by simonihmig** [comment]
A fresh install will use whatever the latest version of `ini` is. And the issue how lock files are handled is everywhere the same and out of scope for this addon, so closing this here.

Updating transitive dependencies with yarn happens to be an issue all the time. Whenever I need to make sure all transitive dependencies of a package are updated, often times `yarn remove package && yarn add -D package` is the only way. Or you could try `resolutions`.

**Comment by cah-briangantzler** [comment]
I apologize for not understanding how this works.

I did not realize that doing  a remove and then an add would change things. I figured this would end up that same.

I do not know why yarn did not accurately list who was hoisting ini, but after I removed ember-bootstrap, it then said that ember-cli was the culprit, after removing and adding ember-cli, then re-adding ember-bootstrap I now have the correct version of ini.

Thanks for the info and hope this may help anyone else trying to figure this out.
